ama-cli/settings.gradle

```diff
@@ -13,25 +13,4 @@
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
- */
-package org.apache.amaterasu.leader.yarn
-
-import org.apache.amaterasu.leader.common.launcher.AmaOpts
-import org.apache.amaterasu.leader.common.launcher.ArgsParser
-import org.apache.amaterasu.leader.common.utilities.MessagingClientUtil
-
-class AppMasterArgsParser: ArgsParser() {
```

Comment:
This is YARN specific, the CLI should have it's own parser

Reply:
it is not here any more, isnt it?

Reply:
BTW i am using a libary argparser for Kotlin CLI